config/config.yaml

```diff
@@ -0,0 +1,29 @@
+version: 0.1
```

Comment:
it seems that the configuration file is not needed if you just would like to enable ReportCaller for logrus

Reply:
removed